src/ert/experiment_server/_server.py

```diff
@@ -121,11 +128,11 @@ async def _server(self) -> None:
     # pylint: disable=line-too-long
     def add_legacy_experiment(  # pylint: disable=too-many-arguments
         self,
-        ert: EnKFMain,
+        ert: "EnKFMain",
         ensemble_size: int,
         current_case_name: str,
         args: Any,
-        factory: Callable[[EnKFMain, int, str, Any, str], Experiment],
+        factory: Callable[["EnKFMain", int, str, Any, str], Experiment],
         experiment_id: str,
     ) -> str:
         """add_legacy_experiment(self, ert, ensemble_size: int,current_case_name: str, factory: Callable[[Any, int, str, Any, str], Experiment], experiment_id: str)
```

Comment:
this line should be removed, and then we can probably remove the pylint disable on line 128